packages/engine/lib/orchestrator/src/process/local.rs

```diff
@@ -41,6 +47,8 @@ impl process::Process for LocalProcess {
             })
             .wrap_err("Could not kill the process")?;
```

Comment:
I think we should run the cleanup regardless if we failed to kill the process.

Reply:
Seems fair https://github.com/hashintel/hash/pull/477/commits/270e45fbe5109ed3b33a46d0ebf52538d20503bd